README.md

```diff
@@ -5,6 +5,68 @@ https://flutter.dev/
 Flutter in an open Source SDK for creating high-performance mobile apps for IOS and Android.
 The Flutter makes it easier for you to build user interfaces, while reducing the amount of code required to create and update your app.
 
+## Why use Flutter?
+
+- Flutter can be used to build applications for Android and IOS
+- The Dart language used in Flutter is quite similar to Java
+- We can update the application in real time
+- If we close the application when we open it again we can continue from where we were
+- Flutter benefits from several objects that help us to facilitate the construction process
+
+## Core Principles
+
+Flutter is designed for the creation of 2D mobile applications.</br> 
+It contains several widgets and tools that facilitate the realization of an app.</br>
+
+## Widgets
+
+Widget is a component that can be used to simplify the process of building applications.</br>
+It usually contains windows, buttons, icons, menus, scroll bars and other features.</br>
+
+A widget can define:
```

Comment:
```suggestion
A widget can be defined as:
```

Reply:
I still have to practice a lot of English 😄